src/goDebugConfiguration.ts

```diff
@@ -82,6 +89,9 @@ export class GoDebugConfigurationProvider implements vscode.DebugConfigurationPr
 		if (!debugConfiguration.hasOwnProperty('showGlobalVariables') && dlvConfig.hasOwnProperty('showGlobalVariables')) {
 			debugConfiguration['showGlobalVariables'] = dlvConfig['showGlobalVariables'];
 		}
+		if (!debugConfiguration['cwd']) {
+			debugConfiguration['cwd'] = '${workspaceFolder}';

```

Comment:
this gets passed to delve for `--wd`. I am not sure if this will result in unexpected behavior for existing `launch` requests. I know we need to this to do the mapping between local and remote paths in case of attach/remote. Can we add this only for `attach` requests?